validator/models/validation_run.py

```diff
@@ -140,6 +140,18 @@ class ValidationRun(models.Model):
     # dataset_configurations from DatasetConfiguration
     # celery_tasks from CeleryTask
 
+    @property
+    def job_status(self):
```

Comment:
job is something different than validation - name it either validation_status or simply status